index.js

```diff
@@ -7,6 +7,9 @@ const parseArgs = (
   if (typeof options !== 'object' || options === null) {
     throw new Error('Whoops!')
   }
+  if(options.withValue && !Array.isArray(options.withValue)) {
```

Comment:
are `false` and `0` and `null` and `NaN` and `undefined` and `''` all acceptable values?

Reply:
Hey @ljharb thanks for taking the time to review my PR, appreciate that catch!! I fixed my PR to handle the above cases, but I left undefined as an acceptable value for options.withValue so that options can be an empty object {}.
Thanks again